lib/cogctl/actions/relay_groups/create.ex

```diff
@@ -40,7 +40,7 @@ defmodule Cogctl.Actions.RelayGroups.Create do
 
   defp add_to_group(group, relays, endpoint) do
     Enum.flat_map_reduce(relays, 0, fn(relay, acc) ->
-       case CogApi.HTTP.Client.relay_group_add_relay(%{name: group.name}, %{relay: relay}, endpoint) do
+       case CogApi.HTTP.Client.relay_group_add_relays(%{name: group.name}, %{relays: [relay]}, endpoint) do
```

Comment:
If the API does things atomically, then I think we should just do what the API does. As long as we can give good, actionable error messages, I think we're good.

I think the reason we did things like that was that we didn't actually have an API for doing things all at once.

I think the closer we get to having cogctl just be a minimal shell over the API, the better off we'll be.